modules/sfp_flickr.py

```diff
@@ -144,23 +167,61 @@ def handleEvent(self, event):
 
             self.sf.info("Parsing page " + str(page) + " of " + str(pages))
 
-            # Extract emails
+            # Extract data
             for photo in data['photos']['photo']:
-                emails = self.sf.parseEmails(str(bytes(photo).decode('unicode-escape')))
+                emails = self.sf.parseEmails(str(photo))
                 for email in emails:
-                    # Skip unrelated emails
-                    mailDom = email.lower().split('@')[1]
-                    if not self.getTarget().matches(mailDom, includeChildren=True, includeParents=True):
-                        self.sf.debug("Skipped address: " + email)
+                    if email in self.results:
+                        continue
+
+                    mail_domain = email.lower().split('@')[1]
```

Comment:
Worth trying to handle an exception here in case we get a malformed e-mail address?

Reply:
`email` comes from `self.sf.parseEmails` a few lines above.

Are you suggesting the `self.sf.parseEmails` function can return malfored email addresses, and if so, what validation could be implemented here that would be superior to the validation in `self.sf.parseEmails`, and why should that validation not be applied in `self.sf.parseEmails` instead?